idea/idea-frontend-independent/tests/org/jetbrains/kotlin/test/uitls/IgnoreTests.kt

```diff
@@ -173,9 +173,10 @@ object IgnoreTests {
         const val FIR_COMPARISON_MUTLTILINE_COMMENT = "/* FIR_COMPARISON */"
         const val IGNORE_FIR = "// IGNORE_FIR"
         const val FIX_ME = "// FIX_ME: "
+        const val FIR_IDENTICAL = "// FIR_IDENTICAL"
```

Comment:
@darthorimar What do you think of the usage of `FIR_IDENTICAL` here? For diagnostics, FIR should more or less replicate the FE1.0 behavior. Or do you prefer to use `FIR_COMPARISON` for this case?

Reply:
Yeah, we want to have semantic and error highlighting to be close to FE1.0 behavior, so FIR_IDENTICAL seems appropriate